Library/include/CSP/Systems/ECommerce/ECommerce.h

```diff
@@ -103,6 +102,8 @@ class CSP_API ProductVariantInfo
 	csp::common::Array<VariantOptionInfo> Options;
 	/// @brief Unit price for the variant
 	CurrencyInfo UnitPrice;
+    /// @brief Quantity of variant available
+	int32_t QuantityAvailable;
```

Comment:
One small change request, the [AC](https://magnopus.atlassian.net/browse/OF-1172) asks for the variable exposed via the public API to be named `AvailableStock` - please could we update it to that?